app/javascript/components/users/show/CreateEventPopUp.vue

```diff
@@ -176,8 +185,47 @@
               targets_attributes: this.targets
             }
           }
-        }).then((response) => location.reload())
-          .catch((error) => console.log(error.response));
+        }).then((response) => this.showNotice(response.data))
+      },
+      showNotice(response) {
+        (() => { 
+          if (response.status == 'created' || response.status == 'updated') {
+            return location.reload()
+          }else {
+            this.cleanErrorMessages('.erorrs', '.fields-error');
+
+            for (let error in response) {
+              this.addErrorMessages(error, response[error], 3)
+            }
+          }
+        })();
+      },
+      cleanErrorMessages(errorsMessagesClassName, errorsFieldsClassName) {
+        let errors_messages = document.querySelectorAll(errorsMessagesClassName);
+        let errors_fields = document.querySelectorAll(errorsFieldsClassName);
+
+        if (errors_fields) {
+          [].forEach.call(errors_fields, function(el) {
```

Comment:
What is the purpose iterating on an empty Array?

Reply:
> What is the purpose iterating on an empty Array?

It is an alias for Array.prototype.forEach.call(...);
Short example
```
[1, 2, 3].forEach.call(["a", "b", "c"], function (item, i, arr) {
    console.log(i + ": " + item);
});
```
### Result 
0: "a"
1: "b"
2: "c"